### ISSUE TITLE
Introduce contextual menu

### ISSUE THREAD

**Issue by IAmAnubhavSaini** [IAmAnubhavSaini/jspaint#25]
Introduce only the framework. Actual menu items are out of scope.

**Comment by IAmAnubhavSaini** [comment]
Close after #48 #49 #50 #51 #53 #54 #55 #56.

Basically, as of now, just produce context menu where ever you think will be least frictional. We will later create a special contextual menu popup.

**Comment by IAmAnubhavSaini** [comment]
We have context menu for each tool that we support. So, close it.

Their enhancement is no scope of this issue,
